hs_core/models.py

```diff
@@ -1119,6 +1119,7 @@ def bag_url(cls, resource_id):
     def delete(self, using=None):
         from hydroshare import hs_bagit
         from hs_access_control.models import UserResourcePrivilege, GroupResourcePrivilege
+        from hs_labels.models import UserResourceLabels
```

Comment:
There are other ways to avoid circular imports.

You can avoid them by importing only the modules, but not objects defined within them (like `from hs_labels import models as labels_models` and then referencing `labels_models.UserResourceLabels` within to reference anything from that module. Circular imports only cause problems when you try to reference objects in those modules, by way of `from ... import`, before that module has fully loaded and defined those objects.